near/nep141-locker/src/lib.rs

```diff
@@ -112,11 +118,12 @@ pub trait Prover {
 pub struct Contract {
     pub prover_account: AccountId,
     pub factories: LookupMap<ChainKind, OmniAddress>,
-    pub pending_transfers: LookupMap<Nonce, TransferMessage>,
+    pub pending_transfers: LookupMap<Nonce, TransferMessageStorage>,
     pub finalised_transfers: LookupSet<(ChainKind, Nonce)>,
     pub tokens_to_address_mapping: LookupMap<(ChainKind, AccountId), OmniAddress>,
     pub mpc_signer: AccountId,
     pub current_nonce: Nonce,
+    pub accounts_balances: LookupMap<AccountId, StorageBalance>,
```

Comment:
This storage management mechanism adds quite a bit of complexity. Is this a normal practice for smart contracts on Near? I was thinking if required payments are small enough we can just collect them without refunding

Reply:
Please check these links
https://docs.near.org/concepts/storage/storage-staking
https://nomicon.io/Standards/StorageManagement
https://docs.near.org/tutorials/fts/registering-accounts

Reply:
Also, take into account that these balances will be used not just to pay for storage but also to attach fee in native/base token

Reply:
If refund values are negligible, would it be reasonable to remove this logic to reduce contract complexity?